src/zoid/button/container.jsx

```diff
@@ -59,14 +59,16 @@ export function containerTemplate({ uid, props, tag, context, frame, prerenderFr
     };
 
     const element = (
-        <div id={ uid } onRender={ setupAutoResize } class={ `${ tag } ${ tag }-context-${ context } ${ tag }-label-${ label }` }>
+        <div id={ uid } onRender={ setupAutoResize } class={ `${ tag } ${ tag }-context-${ context } ${ tag }-label-${ label || 'unknown' }` }>
```

Comment:
The label should always default to something, right? We should never need 'unknown'?

Reply:
done